src/utils/authValidations.ts

```diff
@@ -0,0 +1,10 @@
+export function isValidEmail(email: string) {
+  // eslint-disable-next-line no-useless-escape
+  return /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(email)
```

Comment:
@GustavoFSProg The code checks are pointing to vulnerabilities for this Regex. Maybe we could try another one try another approach instead of using Regex.

If you want you could try how it's done in the MUI. Check it out [here](https://github.com/mui-org/material-ui/blob/e724d98eba018e55e1a684236a2037e24bcf050c/docs/src/pages/premium-themes/onepirate/modules/form/validation.js#L2-L6).